# Commanded position for the actuators 
int32 commanded_position
